_includes/live-demos/annotations/index.js

```diff
@@ -3,8 +3,7 @@ var settings = {
   toolbar: ['annotate-alpha'],
   menubar: false,
   height: '750px',
-  content_css: '//www.tiny.cloud/css/codepen.min.css',
-  content_style: '.mce-annotation { background-color: darkgreen; color: white; }',
+  content_style: '.mce-annotation { background-color: darkgreen; color: white; }' + {{site.liveDemoCSSStyles}},
```

Comment:
This should work fine, but to be on the safe side, maybe add a space here? eg:
```
  content_style: '.mce-annotation { background-color: darkgreen; color: white; } ' + {{site.liveDemoCSSStyles}},

```

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb